src/test/java/cn/edu/thu/tsfile/timeseries/write/ReadPageInMemTest.java

```diff
@@ -0,0 +1,251 @@
+package cn.edu.thu.tsfile.timeseries.write;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import cn.edu.thu.tsfile.common.conf.TSFileConfig;
+import cn.edu.thu.tsfile.common.conf.TSFileDescriptor;
+import cn.edu.thu.tsfile.common.constant.JsonFormatConstant;
+import cn.edu.thu.tsfile.common.utils.Pair;
+import cn.edu.thu.tsfile.common.utils.RandomAccessOutputStream;
+import cn.edu.thu.tsfile.common.utils.TSRandomAccessFileWriter;
+import cn.edu.thu.tsfile.file.metadata.enums.CompressionTypeName;
+import cn.edu.thu.tsfile.file.metadata.enums.TSDataType;
+import cn.edu.thu.tsfile.timeseries.read.query.DynamicOneColumnData;
+import cn.edu.thu.tsfile.timeseries.utils.RecordUtils;
+import cn.edu.thu.tsfile.timeseries.write.exception.WriteProcessException;
+import cn.edu.thu.tsfile.timeseries.write.io.TSFileIOWriter;
+import cn.edu.thu.tsfile.timeseries.write.record.TSRecord;
+import cn.edu.thu.tsfile.timeseries.write.schema.FileSchema;
+
+public class ReadPageInMemTest {
+
+	private String filePath = "TsFileReadPageInMem";
+	private File file = new File(filePath);
+	private TSFileConfig conf = TSFileDescriptor.getInstance().getConfig();
+	private WriteSupport<TSRecord> writeSupport = new TSRecordWriteSupport();
+	private InternalRecordWriter<TSRecord> innerWriter;
+	private FileSchema fileSchema = null;
+
+	@Before
+	public void setUp() throws Exception {
+		file.delete();
+		conf.pageSize = 200;
+		conf.rowGroupSize = 100000;
```

Comment:
A fatal error. Note that, conf is implemented as a single instance, which means if you change the default value in it here, the following JUnit test, even in other test class, will obtain the **changed** configuration.